man/networkctl.xml

```diff
@@ -294,6 +294,29 @@ s - Service VLAN, m - Two-port MAC Relay (TPMR)
         which match the file are reconfigured.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term>
+          <command>netns-create</command>
```

Comment:
Do we still need these commands if `NetworkNamespacePath` creates the network namespace automatically?

Reply:
Maybe useful for `systemd-nspawn`?

Reply:
I'm not sure, I think the existing tools `unshare`, `lsns` and `unmount` cover these use cases well enough. Of course, if you or someone else feels we should have these then by all means lets add them. It doesn't hurt anyone if `networkctl` has a few extra commands.

Reply:
id drop them really.